### ISSUE TITLE
No syntax highlighting without (#)minecraft:xxxx

### ISSUE THREAD

**Issue by mathaym25** [Arcensoth/language-mcfunction#31]
![Screenshot 15-12-2019 à 13h06m59s](https://user-images.githubusercontent.com/35702771/70862372-ceb94000-1f3b-11ea-9535-985eeb0a6a4e.png)
This issue is also valid for the entities (eg: `/summon creeper` instead of `/summon minecraft:creeper`)

And if I am not mistaken, the tags are not supposed to be a different color than the blocks?

**Comment by Arcensoth** [comment]
Both of these highlighting anomalies are "working as intended" for the version-agnostic grammar, which is the current default of the extension. This grammar is unaware of commands and their arguments, and simply attempts to classify tokens based on their contents.

The reason for the particular case mentioned here is that it's practically impossible to differentiate a non-namespaced resource location (block, entity, etc) from other types of tokens, without knowing the command its being used in. The version-agnostic grammar doesn't know what a `setblock` command is, nor does it know what a `summon` command is. The last example there with `#test` is likely being classified as a "fakeplayer" because the grammar doesn't know what a `scoreboard` command is and `#` is a character used commonly to prefix fakeplayer names.

On the other hand, the version-specific grammars are the opposite: they are generated based on the command tree for a particular version (via the game's `commands.json` report) and have baked-in support for all of the commands and their arguments. The reason these grammars are not the default is because they are still WIP and most of the argument parsers are incomplete. At the moment of writing they have partial support for `execute` commands and a handful of argument types. You can try out the current 1.15 snapshot version by switching the language to `mcfunction-snapshot` if you wish to help test and provide feedback.

See also #18 for a similar issue.

**Comment by Arcensoth** [comment]
TLDR; This is working as intended for the version-agnostic grammar `mcfunction` due to the flexible yet imprecise nature of grammar. The version-specific grammars (such as `mcfunction-15` and `mcfunction-snapshot`) will support the desired and accurate argument-specific highlighting in full.
